src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -728,11 +728,8 @@
     <value>Open Settings</value>
     <comment>This is a call-to-action hyperlink; it will open the settings.</comment>
   </data>
-  <data name="CommandPalette_OneMatchAvailable" xml:space="preserve">
-    <value>One suggestion available</value>
-  </data>
-  <data name="CommandPalette_ManyMatchesAvailable" xml:space="preserve">
-    <value>{0} suggestions available</value>
-    <comment>{0} will be replaced with a number bigger than 1</comment>
+  <data name="CommandPalette_MatchesAvailable" xml:space="preserve">
+    <value>Suggestions found: {0}</value>
+    <comment>{0} will be replaced with a number.</comment>

```

Comment:
I love this idea @DHowett! It's not perfect, but it's a brilliant interim solution.